stream-chat-android-client/src/main/java/io/getstream/chat/android/client/plugin/Plugin.kt

```diff
@@ -411,9 +411,13 @@ public interface Plugin :
         /* No-Op */
     }
 
-    public fun onUserSet(user: User)
+    public fun onUserSet(user: User) {
+        /* No-Op */
+    }
 
-    public fun onUserDisconnected()
+    public fun onUserDisconnected() {
+        /* No-Op */
+    }
```

Comment:
Why do we need to implement it now on the interface?
It shouldn't be implemented here because it is normally a methdo needed to be implemented on the plugin subclass (They are used to init/destroy the plubing itself).
If the new plugin doesn't need it because there isn't an initialization needed, it should impelement it as "empty method", but keeping the interface without default implementation

Reply:
As I said, They are important methods to initialize/destroy elements within the plugin.
On the case of `ThrottlePlugin` those method should be used to init/clean internal maps used within the plugin

Reply:
I'm not saying they're not important. They should be implemented optionally.
For example, the `onCreate` function of an `Activity` is one of the most important and widely used functions in Android.
Despite this, its implementation is optional.